README.md

```diff
@@ -16,60 +17,52 @@ For contract deployment:
 ## Setup
 
 ```bash
-# init and start podman machine
-podman machine init
-podman machine start
-
-# create template yaml
-# NOTE - replace the executor_local_account with your own account address
-podman run --rm \
-  -e pod_name=astria_stack \
-  -e celestia_home_volume=celestia-home-vol \
-  -e metro_home_volume=metro-home-vol \
-  -e executor_home_volume=executor-home-vol \
-  -e relayer_home_volume=relayer-home-vol \
-  -e conductor_home_volume=conductor-home-vol \
-  -e executor_local_account=0xb0E31D878F49Ec0403A25944d6B1aE1bf05D17E1 \
-  -e celestia_app_host_port=26657 \
-  -e bridge_host_port=26659 \
-  -e sequencer_host_port=1318 \
-  -e sequencer_host_grpc_port=9100 \
-  -e executor_host_http_port=8545 \
-  -e executor_host_grpc_port=50051 \
-  -e scripts_host_volume="$PWD"/container-scripts \
-  -v "$PWD"/templates:/data/templates \
-  dcagatay/j2cli:latest \
-  -o /data/templates/astria_stack.yaml \
-  /data/templates/astria_stack.yaml.jinja2
-
-# run pod
-podman play kube --log-level=debug templates/astria_stack.yaml
-
-# deploy test contract
-git clone https://github.com/joroshiba/test-contracts
-cd test-contracts
-export PRIV_KEY=123...
-RUST_LOG=debug forge create --private-key $PRIV_KEY src/Storage.sol:Storage
-
+# create control cluster
+just create-cluster
+
+# ingress
+just deploy-ingress-controller
+# wait for ingress.
+# NOTE: this may fail quickly with "error: no matching resources found".
+#  please retry the command
+just wait-for-ingress-controller
+
+# deploy
+just deploy-astria
 ```
 
+### Connecting Metamask
```

Comment:
I tried this step immediately after the previous command but got the error `Could not fetch chain ID. Is your RPC URL correct?`, looking at the logs-astria, it seemed the containers hadn't started yet. maybe add a note mentioning that?

Reply:
Good catch, I added a `wait-for-astria` for now. I'd like to find a better solution instead of adding another command, but this should work for now.